<launch>

<node pkg="autonomy" name="distance" type="distanceDriver.py"/>
<node pkg="autonomy" name="lines" type="lineDriver.py"/>
<node pkg="autonomy" name="servos" type="servoDriver.py"/>
<node pkg="autonomy" name="core" type="core.py"/>
<node pkg="autonomy" name="motorDriver" type="motorDriver.py"/>
<include file="$(find raspicam_node)/launch/camerav2_1280x960_10fps.launch">
	<arg name="enable_raw" value="true"/>
</include>
</launch>

